### DISCUSSION TITLE
Is there a way to update the cache without using graphql?

### DISCUSSION THREAD

**Discussion by narkeeso** [urql-graphql/urql#2376] [Q&A]
I have a standard REST endpoint that updates a resource but it looks like urql doesn't have a way to update the cache without using a graphql query/mutation. Am I going to have to wrap this REST endpoint in a resolver to get this to work?

**Comment by solomon23** [comment]
We ran into the same issue - have a rest api and a graphql api.  Would be nice if we could clear the document cache of types with an api call

**Comment by villesau** [comment]
For me invalidating part of the cache from outside would probably be enough (something like: `client.invalidateCache(...)`). We, too, have some rest endpoints that have side effects on graphql results and it is not practical to rewrite everything in graphql in a big bang.
